app/src/pages/project/ProjectSessionsPage.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Suspense } from "react";
+import { PreloadedQuery, usePreloadedQuery } from "react-relay";
+import { Outlet } from "react-router";
+
+import { Loading } from "@phoenix/components";
+import {
+  ProjectPageQueriesSessionsQuery,
+  useProjectPageQueryReferenceContext,
+} from "@phoenix/pages/project/ProjectPageQueries";
+import { SessionSearchProvider } from "@phoenix/pages/project/SessionSearchContext";
+import { SessionsTable } from "@phoenix/pages/project/SessionsTable";
+import { SpanFilterConditionProvider } from "@phoenix/pages/project/SpanFilterConditionContext";
+import { TracingRoot } from "@phoenix/pages/TracingRoot";
+
+import { ProjectPageQueriesSessionsQuery as ProjectPageSessionsQueryType } from "./__generated__/ProjectPageQueriesSessionsQuery.graphql";
+
+function SessionsTabContent({
+  queryReference,
+}: {
+  queryReference: PreloadedQuery<ProjectPageSessionsQueryType>;
+}) {
+  const data = usePreloadedQuery(
+    ProjectPageQueriesSessionsQuery,
+    queryReference
+  );
+  return (
+    <SessionSearchProvider>
+      <SessionsTable project={data.project} />
+    </SessionSearchProvider>
+  );
+}
+
+export const ProjectSessionsPage = () => {
+  const { sessionsQueryReference } = useProjectPageQueryReferenceContext();
```

Comment:
huh, this feels slightly off but I don't know of another solution so I guess we must do it. I guess in theory this would be loaded on the router but this is technically maybe faster...